terraform/aws/environments/prod/1-network/files/dns/named.conf.options

```diff
@@ -0,0 +1,20 @@
+options {
+    directory "/var/cache/bind";
+    forward only;
+  # This is the Amazon DNS server, which is always at the VPC's '+2' address
+    forwarders {
+        <add_ip>;
+    };
+    dnssec-validation yes;
+
+    auth-nxdomain no;    # conform to RFC1035
+    listen-on-v6 { any; };
+
+    allow-query { any; };
+};
+# This is a private zone served by Route 53 that uses the Amazon DNS server
+zone "internal.test.cloud.mattermost.com." {
+    type forward;
+    forward only;
+    forwarders { <add_ip>; };
```

Comment:
<add_ip> is ok? or do we need to have an ip? or this will be added in the private?

Reply:
It is in private. This is the IP a user can choose to use for dns queries. Lets keep it generalised here 
for community.